condor/migration-guide.md

```diff
@@ -0,0 +1,193 @@
+- A **V1 transaction**, representing the new standard moving forward.  
+
+Secondly, Casper 2.0 includes new methods, renamed methods and methods with a change in response format. 
+
+At the same time, Casper 1.x deploys will continue to be accepted by Casper 2.0, hence if your entire integration comprises only the submission of Deploys (such as transfers in and out of accounts), you may want to take into account the below considerations prior to Mainnet activation of Casper 2.0. 
+
+### Preparing for Casper 2.0: Compatibility Considerations  
+
+Exchanges **may not need to make immediate changes** before the mainnet activation of **Casper 2.0**; however, this depends on how they interact with the network.  
+
+While the **account-put-deploy** JSON-RPC method continues to accept the same input structure as in **1.x**, the response format of **info_get_deploy** is **not backwards compatible** with **1.x**.  
+
+If an exchange relies on the results of **info_get_deploy**, updates will be required to handle the new response format properly.  
+
+Using 1.x Deploys in a 2.x Node:
+
+   - Casper 2.0 introduces a new Transaction model (replacing the old “Deploy”), but it will continue to accept and execute valid 1.x deploys for now​.
+   
+   - Casper Sidecar's `account_put_deploy` will be compatible with 1.x node `account_put_deploy`
+   
+   - This means the sidecar’s deploy submission method (`account_put_deploy`) remains usable with Casper 2.0 nodes, preserving compatibility for existing integration code. (In the Casper 2.0 JSON-RPC, this method is formally renamed to `account_put_transaction`​)
+   
+   - However, Casper Sidecar's `info_get_deploy` isn't 100% compatible with 1.x nodes `info_get_deploy`. 
+
+Therefore, Exchanges should review how they retrieve deploys/transactions and be aware of the response schema changes when querying for transaction status or details. For schema changes please refer the link [here](https://docs.casper.network/condor/jsonrpc#json-rpc-schema-definitions). 
+
+
+>[!IMPORTANT]
+> **Deprecation of 1.x Structures (e.g. Deploys)**
+> 
+> **1.x structures, such as Deploys, are being deprecated.** While they will continue to function in **Casper 2.0**, they are scheduled to be phased out.  
+> 
+> Exchanges should be aware of this evolution and prepare for the transition towards **TransactionV1** as the go-forward model.
+
+#### **Action Required**  
+
+➡️ Exchanges should review their integration and assess whether their systems depend on **info_get_deploy** results. If so, necessary updates must be made to ensure compatibility with **Casper 2.0**.
+
+➡️ To assess your next steps, please evaluate your integration against the 1.x -> 2.0 JSON RPC changes detailed in the link [here](https://docs.casper.network/condor/jsonrpc#json-rpc-schema-definitions). 
+
+Please refer to this [page](https://docs.casper.network/condor/jsonrpc) for more details on **casper-sidecar** and **JSON RPC** changes.
```

Comment:
The RPC integration section doesn't provide much information about what has to be changed, essentially sending the reader to check the RPC schema by themselves. 

The above has to be rewritten to include the list of changes in the RPC protocol divided into two parts:
1. Change in types with correspondences between the semantically identical types (`Deploy` and `TransactionV1`, `Block` and `BlockV2`)
2. Changes in the RPC methods that clearly describe which methods are deprecated and what are their newer alternatives.

Reply:
Additionally, this section is a very heavy lift. Do we know of case where somebody integrates with the RPC interface directly w/o an SDK?